### PR TITLE
ISPN-16827 Set fixed ports in the UDP stack

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.redhat.com/browse/ISPN-16827

### CODE REVIEW

**Comment by pruivo** [comment]
I tried to speedup the test suite by adding a zero timeout to the first node starting... it saves 3 min when compared with the main branch. Not sure if it is worth keeping it and I'll let the reviewer decide :)

**Review by wburns — APPROVED** [review]
LGTM, not sure if others want to chime in or not.

File: documentation/src/main/asciidoc/topics/ref_jgroups_ports.adoc
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/ref_jgroups_ports.adoc:22]:
```suggestion
|Failure detection is provided by `FD_SOCK2`.
```

**Review by theashiot — COMMENTED** [review]
Added a minor suggestion, lgtm otherwise!

thanks,

ashwin

**Comment by pruivo** [comment]
It is updated with Ashwin's suggestion.

**Comment by wburns** [comment]
Integrated into main, thanks @pruivo !

**Pull request merged**
